definitions/npm/react-native-vision-camera_v2.x.x/flow_v0.142.x-/react-native-vision-camera_v2.x.x.js

```diff
@@ -0,0 +1,1358 @@
+declare module "react-native-vision-camera" {
+  // import type { ViewProps } from "react-native";
```

Comment:
React native is flow-typed but not available in the test environment.. We should probably aim to add this into the harness so that react-native packages can resolve properly

Reply:
Yeah; importing from react-native would be nice. I think that the react-native dependency would still be impossible in flow-typed v4, since react-native types aren't supplied through this project.

Reply:
I'll raise an issue to track, I think I can do something with the CLI later so that we can test with react-native as a dependency

Reply:
@joshuanapoli I need some help. How does `react-native` ship flow definitions? As I was creating the change i realised that there are no `.js.flow` files as far as I can tell in the `react-native` package

Reply:
@Brianzchen The source files in the shipped package are all flow-typed. As far as I understand, the package.json refers to these flow-typed files. As configured by package.json, the flow-typed files will be loaded (and there are not pure JavaScript files at all). Also, configured by package.json, Typescript .d.ts files are loaded from the types directory.

Each of the main sources have a flow comment:

```
/**
 [...]
 * @format
 * @flow strict-local
 */
```

Evidently, the metro bundler strips the flow types.

I hope that this information is correct. It seems surprising to me.